person.rb

```diff
@@ -0,0 +1,25 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = 1
```

Comment:
- [Optional] The way you implemented line 6, all persons will have an id of 1. The id should be unique to each person. Kindly consider making id unique to each person. You can use  this `@id = Random.rand(1..1000)` which generates a random number between 1 and 1000.